pkg/maps/lbmap/maglev_test.go

```diff
@@ -63,6 +63,14 @@ func (s *MaglevSuite) SetUpSuite(c *C) {
 	// Otherwise opening the map might fail with EPERM
 	err = unix.Setrlimit(unix.RLIMIT_MEMLOCK, &tmpLim)
 	c.Assert(err, IsNil)
+
+	Init(InitParams{
```

Comment:
Do we want to move the initialization of maps to `SetupTest` in case the individual test cases manipulate maps?

Reply:
No, I don't think so because the previous behavior would be that the map objects are initialized by virtue of being package-level variables, in other words, they'd be initialized once. Putting this call here now has the same effect of initializing the objects once.